models/users.js

```diff
@@ -360,6 +360,8 @@ const getRdsUserInfoByGitHubUsername = async (githubUsername) => {
  * @param {Array} query.levelNumber - Array of levelNumbers to filter the users on
  * @param {Array} query.tagId - Array of tagIds to filter the users on
  * @param {Array} query.state - Array of states to filter the users on
+ * @param {String} query.role - filter the users on
+ * @param {String} query.verified - filter the users on
```

Comment:
What does this mean? I guess you forgot completing the sentence 😂

Reply:
yes , thought would complete it later